docs/source/overview/quicktour.md

```diff
@@ -304,3 +216,5 @@ from olive.engine.footprint import Footprint
 footprint = Footprint.from_file("footprints.json")
 footprint.plot_pareto_frontier()
 ```
+### Olive Packaging
+Olive also can package output artifacts when user adds `PackagingConfig` to Engine configurations. Please refer to [Packaing Olive artifacts](../tutorials/packaging_output_models.md) for more details.
```

Comment:
We should include packaging config in this quick tour example.

Reply:
included a config exmaple and a result folder structure example